server/migrations/0001-comment-status.js

```diff
@@ -0,0 +1,12 @@
+import {Comment} from '/lib/documents/comment';
+
+class Migration extends Document.AddRequiredFieldsMigration {
```

Comment:
I `master` branch I added [this plugin](https://github.com/babel/babel/tree/master/packages/babel-plugin-proposal-class-properties) so that we can write:

```js
class Migration extends Document.AddRequiredFieldsMigration {
  name = "Adding status field"
  fields = {
    status: Comment.STATUS.CREATED,
  }
}
```